### ISSUE TITLE
Tier widget

### ISSUE THREAD

**Issue by alanna** [opencollective/opencollective#2169]
## User story 
As a core contributor, I want to embed a tier widget on my webpage, so supporters can click on it and jump directly into the contribution flow. I also want it to display how many financial contributors that tier has.

(This is a user requested feature)

**Comment by znarf** [comment]
What should the widget do? Can't they just have link?

**Comment by alanna** [comment]
Yes I suggested to them to just make a link using text or image of the tier box once we have the new individual pages for tiers released. They want it to show the number of financial contributors at that tier, which is a dynamically updating number, hence the request for a widget.

**Comment by Betree** [comment]
Related to https://github.com/opencollective/opencollective/issues/2230
